src/test/java/org/folio/marc4ld/mapper/field100and700/Marc2Bibframe100And700IT.java

```diff
@@ -0,0 +1,81 @@
+import static org.folio.ld.dictionary.PropertyDictionary.AUTHORITY_LINK;
+import static org.folio.ld.dictionary.PropertyDictionary.CONTROL_FIELD;
+import static org.folio.ld.dictionary.PropertyDictionary.DATE;
+import static org.folio.ld.dictionary.PropertyDictionary.EQUIVALENT;
+import static org.folio.ld.dictionary.PropertyDictionary.FIELD_LINK;
+import static org.folio.ld.dictionary.PropertyDictionary.LINKAGE;
+import static org.folio.ld.dictionary.PropertyDictionary.NAME;
+import static org.folio.ld.dictionary.PropertyDictionary.NAME_ALTERNATIVE;
+import static org.folio.ld.dictionary.PropertyDictionary.NUMERATION;
+import static org.folio.ld.dictionary.PropertyDictionary.TITLES;
+import static org.folio.ld.dictionary.ResourceTypeDictionary.WORK;
+import static org.folio.marc4ld.mapper.test.TestUtil.loadResourceAsString;
+import static org.folio.marc4ld.mapper.test.TestUtil.validateEdge;
+import static org.folio.marc4ld.mapper.test.TestUtil.validateResource;
+import static org.folio.marc4ld.test.helper.ResourceEdgeHelper.getEdge;
+import static org.folio.marc4ld.test.helper.ResourceEdgeHelper.getWorkEdge;
+
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import org.folio.ld.dictionary.PredicateDictionary;
+import org.folio.ld.dictionary.ResourceTypeDictionary;
+import org.folio.ld.dictionary.model.Resource;
+import org.folio.marc4ld.Marc2LdTestBase;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.CsvSource;
+
+class Marc2Bibframe100And700IT extends Marc2LdTestBase {
+
+  @ParameterizedTest
+  @CsvSource(value = {
+    "PERSON, CREATOR, fields/100_700/marc_100_person.jsonl",
+    "PERSON, CONTRIBUTOR, fields/100_700/marc_700_person.jsonl",
+    "FAMILY, CREATOR, fields/100_700/marc_100_family.jsonl",
+    "FAMILY, CONTRIBUTOR, fields/100_700/marc_700_family.jsonl"
+  })
+  void shouldMapFields100And700(ResourceTypeDictionary resourceType, PredicateDictionary predicate, String marcFile) {
+    // given
+    var marc = loadResourceAsString(marcFile);
+
+    // when
+    var resource = marcBibToResource(marc);
+
+    assertThat(resource).isNotNull();
+    var work = getWorkEdge(resource).getTarget();
+    assertThat(work)
+      .satisfies(w -> validateResource(w, List.of(WORK), new HashMap<>(), ""))
+      .satisfies(w -> validateEdges(w, resourceType, predicate));
+  }
+
+  private void validateEdges(Resource work, ResourceTypeDictionary resourceType,  PredicateDictionary predicate) {
+    assertThat(getEdge(work, resourceType))
+      .isPresent()
+      .get()
+      .satisfies(edge -> validateEdge(edge, predicate, List.of(resourceType), getProperties(),
+        "numeration, name, titles, another titles, name alternative, date"));
+  }
+
+  private Map<String, List<String>> getProperties() {
+    return Map.ofEntries(
+      entry(AUTHORITY_LINK.getValue(), List.of("authority link", "another authority link")),
```

Comment:
I've noticed that in some tests we write like this - ```AUTHORITY_LINK.getValue()``` and in some others like this - ```"http://bibfra.me/vocab/lite/authorityLink"```
maybe we should stick to one option?

Reply:
agree, we can discuss it, for me better option is enum usage

Reply:
Hi @AndreiBordak,
It is better to use "http://bibfra.me/vocab/lite/authorityLink". We had a discussion on this few months back.

One key reason is that if a developer accidentally changes the enum's value, the tests will still pass. However, applications that consume the graph (such as the UI) would fail, as they expect the original URI.

Reply:
Done.